src/components/TokenLocking/BoostMeter.tsx

```diff
@@ -21,7 +21,12 @@ export const BoostMeter = ({
 
   const now = useMemo(() => getCurrentDays(startTime), [startTime])
 
-  const value = ((SEASON2_START - now) / SEASON2_START) * 100
+  const currentTimeFactor = getTimeFactor(now)
+  const maxTimeFactor = getTimeFactor(0)
+  const minTimeFactor = getTimeFactor(SEASON2_START)
+
+  // Boost meter values are between 10 and 90
+  const value = ((currentTimeFactor - minTimeFactor) / (maxTimeFactor - minTimeFactor)) * 100
```

Comment:
Can we not simplify this to:
```suggestion
  const value = currentTimeFactor * 100
```

As the time factor is always between 0 and 1?